src/app/portlets/dot-edit-page/layout/dot-additional-actions/dot-additional-actions-menu/dot-legacy-additional-actions-menu.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { MenuItem } from 'primeng/primeng';
+import { MessageService } from '../../../../../api/services/messages-service';
+
+
+@Component({
+    selector: 'dot-legacy-addtional-actions-menu',
+    templateUrl: './dot-legacy-additional-actions-menu.component.html'
+})
+export class DotLegacyAdditionalActionsMenuComponent implements OnInit {
+    @Input() templateId: string;
+    items: MenuItem[];
+
+    constructor(private messageService: MessageService) {
+
+    }
+
+    ngOnInit() {
+        const keys = [
+            'template.action.additional.permissions',
+            'template.action.additional.history',
+            'template.action.additional.properties'
+        ];
+
+        this.messageService.getMessages(keys).subscribe(messages => {
+            this.items = [
+                {
+                    label: messages['template.action.additional.properties'],
+                    url: `/#/edit-page/template/${this.templateId}/properties`
```

Comment:
I think there is a way to navigate to relative routes so you don't have to pass the whole url, check Angular Routing or the doc in PrimeNG Menu.